packages/docusaurus-preset-bootstrap/src/index.js

```diff
@@ -8,6 +8,6 @@
 module.exports = function preset(context, opts = {}) {
   return {
     themes: [['@docusaurus/theme-bootstrap', opts.theme]],
-    plugins: [['@docusaurus/plugin-content-pages', opts.pages]],
+    plugins: [['@docusaurus/plugin-content-blog', opts.blog]],
```

Comment:
We shouldn't remove this, we still need the pages plugin! We should be adding the blog plugin on top of the existing pages plugin.